app/models/ability.rb

```diff
@@ -0,0 +1,18 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    if user.present?
+      can %i(create read update destroy), Journal
+      can %i(create read update destroy), JournalEntry
+      can %i(create show new), User
+      can :update, User, id: user.id
+      cannot :index, User
+      if user.admin?
```

Comment:
```suggestion
    elsif user.admin?
```

Reply:
But a user can be present and admin at the same time, no?

Reply:
I corrected it